### PR TITLE
Proofread the mqtt readme (EN)

### LINKED ISSUE #20: Add into the documentation a destroy() method
This methods ware introduced in: https://github.com/tarantool/mqtt/commit/4adefc7bc45aaae776a677bbb0b54ecf7ec293f4,
https://github.com/tarantool/mqtt/commit/bfc0b20320f9fb2912272ee5fd7fe27f8e42005b

### PR DESCRIPTION
**Pull request by sofiayem**
Fixes #20 and #574 community doc issue.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -455,42 +508,44 @@ Lua API documentation
...
+TODO: describe me.
+
 [Back to content](#content)
 
 ## Examples
 -----------
 
-  Connecting to the MQTT broker
-    * examples/connect.lua
+  The [examples/connect.lua](../examples/connect.lua) file shows how to connect
```
Comment (Totktonada) [inline README.md]: 404. `examples/producer_consumer_queue.lua` below too. I think it is because of `../`.

File: README.md [hunk 2]
```diff
@@ -455,42 +508,44 @@ Lua API documentation
...
-    * on_subscribe
+  * on_publish
 
-    * on_unsubscribe
+  * on_subscribe
 
-    You can find Detailed documentation of those functions at file: mqtt/init.lua
+  * on_unsubscribe
+
+  See the detailed documentation of these functions in the [mqqt.init.lua](../mqqt/init.lua) file.
```
Comment (Totktonada) [inline README.md]: 404.

File: README.md [hunk 3]
```diff
@@ -500,16 +555,17 @@ NOTE
 
 [Back to content](#content)
 
-##See also
+## See also
 ----------
 * [Tarantool](http://tarantool.org) homepage.
```
Comment (Totktonada) [inline README.md]: BTW, should we update the link here to tarantool.io (and with https)?

File: README.md [hunk 4]
```diff
@@ -57,16 +63,21 @@ Lua API documentation
 ### lib_destroy
 ---------------
 
-  Cleanup everithing.
-  Note. you have to call it manually, the module does not use lua's GC.
-  Do not call this untill you call `destroy` for each mqtt object.
+  Cleanup everything.
+
+  Note: The module does not use the Lua's GC and have to be called manually.
```
[the remark this answers is not in the document]
Comment (sofiayem) [reply README.md]: Resolved in #25

File: README.md [hunk 5]
```diff
@@ -350,23 +392,27 @@ Lua API documentation
...
 --------------------
 
-  Configure verification of the server hostname in the server certificate. If
-  value is set to true, it is impossible to guarantee that the host you are
-  connecting to is not impersonating your server. This can be useful in
-  initial server testing, but makes it possible for a malicious third party to
-  impersonate your server through DNS spoofing, for example.
-  Do not use this function in a real system. Setting value to true makes the
-  connection encryption pointless.
+  Configure server hostname verification in the server certificate. If the
```
[the remark this answers is not in the document]
Comment (sofiayem) [reply README.md]: Resolved in #25

File: README.md [hunk 6]
```diff
@@ -350,23 +392,27 @@ Lua API documentation
...
-  value is set to true, it is impossible to guarantee that the host you are
-  connecting to is not impersonating your server. This can be useful in
-  initial server testing, but makes it possible for a malicious third party to
-  impersonate your server through DNS spoofing, for example.
-  Do not use this function in a real system. Setting value to true makes the
-  connection encryption pointless.
+  Configure server hostname verification in the server certificate. If the 
+  `value` parameter is set to `true`, connection encryption is pointless and 
+  it is impossible to guarantee that the host you are connecting to is not 
+  impersonating your server. This can be useful during the initial server
```
[the remark this answers is not in the document]
Comment (sofiayem) [reply README.md]: Resolved in #25

File: README.md (hunk 2 above)
Comment (sofiayem) [reply README.md]: Resolved in #25

File: README.md (hunk 3 above)
Comment (sofiayem) [reply README.md]: Resolved in #25

File: README.md (hunk 1 above)
Comment (sofiayem) [reply README.md]: Resolved in #25

**Pull request merged**